lib/flutter_aira.dart

```diff
@@ -14,7 +14,7 @@ export 'src/models/minute_sharing.dart' show MinuteSharingMember, MinuteSharingI
 export 'src/models/service_request.dart' show ServiceRequest, ServiceRequestState;
 export 'src/models/session.dart' show Session;
 export 'src/models/track.dart' show Track;
-export 'src/models/user.dart' show User;
+export 'src/models/user.dart' show User, LinkedAccount;
```

Comment:
We now need to have more information to know which account is linked. We currently only handle Lyft, but in the past there was Uber as well... We may need to refactor this, but the backend work that way currently and I thought it was ok to leave it as is for the moment.